docs/ORTModule_Training_Guidelines.md

```diff
@@ -0,0 +1,207 @@
+# ONNX Runtime Training Guidelines
+
+## 1. Installation
+
+Refer [https://onnxruntime.ai/](https://onnxruntime.ai/) to download training wheel.
+
+## 2. Use `ORTModule` to Accelerate Forward/Backward
+
+Plug in your `torch.nn.Module` model with `ORTModule` to leverage ONNX Runtime fast training engine.
+
+Sample usage as below:
+```diff
+	model = build_model()
+
++	from onnxruntime.training import ORTModule
++	model = ORTModule(model)
```

Comment:
nit: do we want to point out the passed arg "model" is before deepspeed wrapping?

Reply:
Yeah, I think we can make it clear if we used to have issues when putting ORTModule after DeepSpeed? Have you ever hit any issue? Any suggestion how to describe that?

Reply:
Add a notice as suggestions as we talked offline. Thank you!!